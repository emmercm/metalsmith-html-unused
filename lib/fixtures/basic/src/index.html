<!DOCTYPE html>
<html lang="en">
<head>
  <link rel="manifest" href="manifest.json">
  <link rel="stylesheet" href="static/style.css">
</head>
<body>
<script src="static/script.js"></script>
</body>
</html>
