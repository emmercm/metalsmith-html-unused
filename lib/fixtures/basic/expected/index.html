<!DOCTYPE html>
<html lang="en">
<head>
  <link rel="manifest" href="manifest.json">
  <link rel="stylesheet" href="style.css">
</head>
<body>
<script src="script.js"></script>
</body>
</html>
